src/components/paragraphs/sum-pill-banner/sum-pill-banner-paragraph.tsx

```diff
@@ -91,29 +81,19 @@ const SumPillBannerParagraph = ({paragraph, ...props}: Props) => {
                 )}
               >
                 {paragraph.sumPillBannerLink.title}
-              </Link>
+              </ActionLink>
             </div>
           )}
         </div>
 
         {paragraph.sumPillBannerCards && (
-          <div className="flex flex-col gap-20 xl:flex-row">
-            {paragraph.sumPillBannerCards.map(card => {
-              const cardBehaviors = getParagraphBehaviors(card)
-              cardBehaviors.su_card_styles = {
-                ...(cardBehaviors.su_card_styles || {}),
-                heading: Element === "div" ? "h2" : "h3",
-                sum_card_variant: "pill",
-                sum_card_bg_color_variant: false,
-              }
-              card.behaviors = JSON.stringify(cardBehaviors)
-              return (
-                <CardParagraph
-                  key={card.id}
-                  paragraph={card}
-                />
-              )
-            })}
+          <div className="cc rs-mb-8 -mt-96 flex w-full flex-col gap-20 lg:flex-row">
+            {paragraph.sumPillBannerCards.map(card => (
+              <CardParagraph
```

Comment:
Should we force the cards to be pills like I had?

Reply:
Oh yes! Most definitely — the chunk of code has been added back in 😌